composer.json

```diff
@@ -29,7 +29,7 @@
         "illuminate/testing": "^7.0",
         "laravel-zero/foundation": "^7.0",
         "league/flysystem": "^1.0.8",
-        "nunomaduro/collision": "^4.1",
+        "nunomaduro/collision": "^4.1||^5.0",
```

Comment:
Maybe we can remove collision 4?

Reply:
I didn't want to remove it incase there are current packages that require it. 🤔 Then we can remove it in `v8` as it'd (kind of?) be a breaking change.

But if you're ok with removing v4, then I don't see why not. 👍

Reply:
I don't see any problem on removing the v4 there.

Reply:
Ah, just realised Collision 5 requires PHP 7.3+, but LZ7 still supports PHP 7.2.5+, so we need to keep Collision 4 support.

Reply:
@nunomaduro, is this ok? 🤔 Or should we just not support Pest yet in LZ7? 🙂

Reply:
Yeah - lets port all of this new stuff  to LZ8